system/security/shared/seclib.hpp

```diff
@@ -216,6 +216,7 @@ interface ISecAuthenticEvents : extends IInterface
 interface ISecProperty : extends IInterface
 {
     virtual const char * getName() = 0;
+    virtual void setValue(const char * value) = 0;
```

Comment:
@timothyklemm can you add the "override" keyword to the implementation of the method in CSecurityResource? I know other methods don't have it, but let's start adding it for any method we touch...

Reply:
I added override to both implementations (CSecurityResource and CLdapSecResource). Both already specified virtual, which I chose not to remove. Both are redundant, but override adds value without conflicting.